packages/components/select-v2/src/select.vue

```diff
@@ -283,6 +283,7 @@ import { ClickOutside } from '@element-plus/directives'
 import ElTooltip from '@element-plus/components/tooltip'
 import ElTag from '@element-plus/components/tag'
 import ElIcon from '@element-plus/components/icon'
+import { useCalcInputWidth } from '@element-plus/hooks'
```

Comment:
Simplifying the code with this hook is a good refactor, but seems unrelated to the issues. How about moving it into separate PR to not mix concerns?